CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **CHANGES**
 
+- Upgrade EFA installer to version 1.10.1
```

Comment:
probably it's worth to list the components even with no changes in version

Reply:
Done